### ISSUE TITLE
ISO_A2 for France is -99 instead of FR

### ISSUE THREAD

**Issue by s-fleck** [nvkelso/natural-earth-vector#284]
concerns version 4.1.0 of at least

ne_50m_admin_0_countries_lakes
ne_50m_admin_0_countries

**Comment by jimutt** [comment]
The same issue applies to  ne_10m_admin_0_countries 4.1.0

**Comment by lsalehar** [comment]
There are a few countries that have this problem: 
- Portugal
- Norway
- France

**Comment by pope1ni** [comment]
These were all the ones I found in `ne_50m_admin_0_countries` with `-99` in one of the fields:

| ISO_A2 | ISO_A3 | ISO_N3 | NAME |
| :-- | :-- | :-- | :-- |
| -99 | -99 | -99 | Somaliland |
| -99 | -99 | -99 | Norway |
| -99 | -99 | 250 | France |
| -99 | -99 | -99 | N. Cyprus |
| -99 | -99 | -99 | Indian Ocean Ter. |
| -99 | -99 | 036 | Ashmore and Cartier Is. |
| -99 | -99 | -99 | Siachen Glacier |
| XK | -99 | -99 | Kosovo |

See also: https://github.com/etpinard/sane-topojson/blob/c2352d6b63600a17810bdde265b178d574c9ed7d/bin/geo2topo.js#L121-L136

**Comment by nvkelso** [comment]
The fix for this is described in https://github.com/nvkelso/natural-earth-vector/issues/268 (really this is a dup of that issue).

**Comment by nvkelso** [comment]
@mathissen and @varyamereon this is the issue you want to follow.

**Comment by nvkelso** [comment]
Fixed via https://github.com/nvkelso/natural-earth-vector/pull/446.

**Comment by JohnRDOrazio** [comment]
This isn't fixed by #446 for Norway, see https://github.com/nvkelso/natural-earth-vector/blob/master/geojson/ne_50m_admin_0_countries.geojson :
```json
            "properties": {
                ...
                "FIPS_10": "-99",
                "ISO_A2": "-99",
                "ISO_A2_EH": "NO",
                "ISO_A3": "-99",
                "ISO_A3_EH": "NOR",
                "ISO_N3": "-99",
                "ISO_N3_EH": "578",
                ...
            },
```

**Comment by aktiur** [comment]
As of 5.1.1, France still has `-99` as `ISO_A2`, same as Norway.
